accepted/2025/mono.md

```diff
@@ -0,0 +1,108 @@
+﻿# Reducing support costs related to Mono runtime
+<!-- Replace `Title` with an appropriate title for your design -->
+
+- Author Name jeffkl
+
+## Summary
+<!-- One-paragraph description of the proposal. -->
+NuGet is already available across platforms since it ships as part of the .NET SDK.
+However, `NuGet.exe` is a .NET Framework application that works on the Mono runtime, making it possible to run on other platforms like Mac and Linux.
+Support for `NuGet.exe` is not officially supported on the Mono runtime, but does require some maintenance overhead since we still run tests against this environment.
+In order to reduce complexity of our build pipelines, we would like to stop running out tests on the Mono runtime.
+
+
+## Motivation
+<!-- Why are we doing this? What pain points does this solve? What is the expected outcome? -->
+The Mono runtime was the original cross-platform implementation of the .NET Framework.
+With the creation of the .NET SDK, which is also cross-platform, it is now highly recommended that users port their applications to .NET Core rather than build .NET Framework applications that require Mono.
```

Comment:
```suggestion
With the creation of the .NET Core, which is also cross-platform, it is now highly recommended that users port their applications to .NET Core rather than build .NET Framework applications that require Mono.
```